### DISCUSSION TITLE
"Send via email" in file context menu

### DISCUSSION THREAD

**Discussion by m5k8** [doublecmd/doublecmd#1346] [Suggestions]
First of all, thank you for this great piece of software. Now, I'm using it exclusively. But there are some pieces missing, that are really useful.
I'd be very grateful for "Send via email" option in file context menu, so the currently selected files in commander could be sent via email, without tedious switching to email client, "attaching", traversing the directory tree to find files, when the file(s) are already found in the commander.
Best regards!

**Comment by alexx2000** [comment]
Under Windows DC has such capability via system context menu. Under Linux [you can add such menu item yourself.](https://doublecmd.h1n.ru/viewtopic.php?p=23542)

**Comment by m5k8** [comment]
Windows is beyond my interest.
It's good that there's capability to add own menu items, but it's not scalable. I can do it once, but having to repeat it many times on several computers is just not comfortable. And I use quite a few of them. I can probably copy some config files, but still, no joy.
I appreciate the possibility to add user items to the menu, that's great for some special ops that user may need for some special files (to him), but "send via email" is just so generic and universally useful that please reconsider doing your users a favor.
